### ISSUE TITLE
Hello! I am using  ........

### ISSUE THREAD

**Issue by zhouxinghong** [gradle/gradle#20537]

**Comment by vlsi** [comment]
@zhouxinghong , please follow https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#getting-started

a) You do not need `buildscript {` block.
b) Please use `plugins { id(...) }` syntax instead of `apply(plugin = ...`
c) Prefer configuring the repositories via `settings.gradle.kts` -> `dependencyResolutionManagement` so they are declared in a single place for all the projects: https://docs.gradle.org/current/userguide/declaring_repositories.html#sub:centralized-repository-declaration
d) Prefer configuring the repositories via `settings.gradle.kts` -> `pluginManagement`: https://docs.gradle.org/current/userguide/plugins.html#sec:custom_plugin_repositories

Basically, "c" would resolve your issue.

**Comment by zhouxinghong** [comment]
Thank you teacher for your careful help! thank you very much! Again, I wish the teacher the best of luck in all aspects! You are great! 👍😜
